### DISCUSSION TITLE
grbl Twotrees Totem S laser engraver (v.4.4.1) not moving

### DISCUSSION THREAD

**Discussion by Merckymerc** [arkypita/LaserGRBL#1561] [Q&A]
Just purchased this laser engraver. Finally got everything installed correctly (I believe). Driver, config file, baud rate, port all working. Custom buttons come up. Lights turn on on the actual laser. It seems to "connect" correctly (it says connected). Laser cross-hairs shine on the surface below the laser.

But the jog buttons create no movement. If an image is loaded, laser head does not move at all (or burn) when I start the job.

Help please!

**Comment by arkypita** [comment]
Not a LaserGRBL issue!
Check motor wiring.
